go.mod

```diff
@@ -7,25 +7,31 @@ require (
 	github.com/davecgh/go-spew v1.1.2-0.20180830191138-d8f796af33cc
 	github.com/ghodss/yaml v1.0.1-0.20190212211648-25d852aebe32 // indirect
 	github.com/go-bindata/go-bindata v3.1.2+incompatible
-	github.com/openshift/api v0.0.0-20210817132244-67c28690af52
+	github.com/go-logr/logr v1.1.0 // indirect
+	github.com/google/go-cmp v0.5.6 // indirect
+	github.com/google/gofuzz v1.2.0 // indirect
+	github.com/openshift/api v0.0.0-20210907212721-c9d6fcc62893
```

Comment:
I'd expect only openshift/api being bumped here. Same comment as with https://github.com/openshift/client-go/pull/194#discussion_r709075079 Also I'd bump client-go as well, when touching it.